skllm/models/gpt/gpt.py

```diff
@@ -66,10 +78,19 @@ def __init__(
         openai_org: Optional[str] = None,
         n_epochs: Optional[int] = None,
         custom_suffix: Optional[str] = "skllm",
+        is_multi_label: Optional[bool] = False,
+        max_labels: int = 3,
```

Comment:
Maybe it is better to have multi label as a separate class just so it is in line with other estimators.

Reply:
Created a new class for multi label